Orange/widgets/data/owfile.py

```diff
@@ -385,19 +385,36 @@ def _describe(self, table):
             descs[0] = "<b>{}</b>".format(descs[0])
         if descs:
             text += "<p>{}</p>".format("<br/>".join(descs))
-
-        text += "<p>{} instance(s), {} feature(s), {} meta attribute(s)".\
-            format(len(table), len(domain.attributes), len(domain.metas))
+        text += "<p>{} instance(s)".format(len(table))
+        if not table.has_missing():
+            text += " (no missing values)"
+        text += "<br/>{} feature(s)".format(len(domain.attributes))
+        if table.has_missing_attribute():
+            miss = "%.1f" % table.get_nan_percentile_attribute()
+            text += " ({}% missing values)".format(miss)
+        else:
+            text += " (no missing values)"
         if domain.has_continuous_class:
-            text += "<br/>Regression; numerical class."
+            text += "<br/>Regression; numerical class"
+            if table.has_missing_class():
+                miss = "%.1f" % table.get_nan_percentile_class()
+                text += " ({}% missing values)".format(miss)
+            else:
+                text += " (no missing values)"
         elif domain.has_discrete_class:
-            text += "<br/>Classification; categorical class with {} values.".\
+            text += "<br/>Classification; categorical class with {} values".\
                 format(len(domain.class_var.values))
+            if table.has_missing_class():
+                miss = "%.1f" % table.get_nan_percentile_class()
+                text += " ({}% missing values)".format(miss)
```

Comment:
Why not something like

    text += " ({:.1f}% missing values)".format(table.get_nan_percentile_class())

Or

    miss = table.get_nan_percentile_class()
    text += " ({:.1f}% missing values)".format(miss)

, so that we don't format the string in two lines in two different ways.